editor/components/page-attributes/order.js

```diff
@@ -39,13 +39,23 @@ export function PageAttributesOrder( { onUpdateOrder, instanceId, order } ) {
 				id={ inputId }
 				size={ 6 }
 			/>
-		</PageAttributesCheck>
+		</Fragment>
+	);
+}
+
+// TODO: Refactor to use a future ifPostTypeSupports HoC
+function PageAttributesOrderWithChecks( props ) {
+	return (
+		<PostTypeSupportCheck supportKeys="page-attributes">
+			<PageAttributesOrder { ...props } />
+		</PostTypeSupportCheck>
 	);
 }
 
 const applyConnect = connect(
 	( state ) => {
 		return {
+			postTypeSlug: getCurrentPostType( state ),
```

Comment:
It's not used in this file. Should we remove it or maybe move to the component that uses it?